stingray/modeling/parameterestimation.py

```diff
@@ -794,3 +877,76 @@ def plotfits(self, res1, res2=None, save_plot=False,
                 plt.savefig(namestr + '_ps_fit.png', format='png')
 
         return
+
+def calc_likelihood(measured_vals, updated_model, weights, x, m=1):
+    pll = PSDLogLikelihoodBound(x, measured_vals, updated_model, m=m)
```

Comment:
This is for the Astropy interface. It needs a function to call with those non-keyword arguments